crates/swift-bridge-ir/src/bridged_type/bridgeable_result.rs

```diff
@@ -460,9 +496,9 @@ impl BuiltInResult {
         let trimmed = trimmed.trim_end_matches(" >");
 
         // [A, B]
-        let mut ok_and_err = trimmed.split(",");
-        let ok = ok_and_err.next()?.trim();
-        let err = ok_and_err.next()?.trim();
+        let ok_and_err = trimmed.rsplit_once(",")?;
```

Comment:
If we want to support returning `Result<Any Type, tuple>` from Rust functions, we need to parse `Result<SomeType, (i32, u32, String)>` or something like that.